src/converters/freetradeConverter.test.ts

```diff
@@ -0,0 +1,164 @@
+import { FreetradeConverter } from "./freetradeConverter";
+import { YahooFinanceService } from "../yahooFinanceService";
+import { GhostfolioExport } from "../models/ghostfolioExport";
+
+describe("freetradeConverter", () => {
+
+  beforeEach(() => {
+    jest.spyOn(console, "log").mockImplementation(jest.fn());
```

Comment:
I saw this change in your other PR, and want to take a better look at this. Can you remove the mock from this PR so it works like the other converter tests for now?